### DISCUSSION TITLE
Question about importing Tiger data

### DISCUSSION THREAD

**Discussion by gpage123** [osm-search/Nominatim#2218] [Q&A]
I'm very new to nominatim. I've stood up a docker based 3.6 version and imported the OSM data with US postal codes since we're interested in the US only. I've imported the 2020 Tiger data but when I spot check against various lat/lon I sometimes get street numbers from my server and other times not, while I consistently get street numbers from the OSM API. An example is:

38.668674,-89.995318

OSM API:
1724, West Slauson Avenue, Harvard Park, Los Angeles, Los Angeles County, California, 90047, United States

My server JSON:

{"place_id":35986209,"licence":"Data © OpenStreetMap contributors, ODbL 1.0. https://osm.org/copyright","osm_type":"way","osm_id":398417482,"lat":"33.98912998169946","lon":"-118.30704669537441","display_name":"West Slauson Avenue, Harvard Park, Los Angeles, Los Angeles County, California, 90230, United States","address":{"road":"West Slauson Avenue","suburb":"Harvard Park","city":"Los Angeles","county":"Los Angeles County","state":"California","postcode":"90230","country":"United States","country_code":"us"},"boundingbox":["33.9891243","33.9891377","-118.3083982","-118.3050133"]}

I could use some help in determining why I'm not getting the street number back from my server. I have added @define('CONST_Use_US_Tiger_Data', true); to local.php.

Thanks

**Comment by mtmail** [comment]
Check the value is `true` in `settings-frontend.php` as well. Related dicussion https://github.com/osm-search/Nominatim/discussions/2176#discussioncomment-395497
